module/os/linux/zfs/zfs_uio.c

```diff
@@ -252,15 +248,15 @@ zfs_uio_prefaultpages(ssize_t n, zfs_uio_t *uio)
 			/* touch each page in this segment. */
 			p = iov->iov_base + skip;
 			while (cnt) {
-				if (copy_from_user(&tmp, p, 1))
+				if (get_user(tmp, (uint8_t *)p))
```

Comment:
Indeed this is the only file with 2 occurrences of `get_user()` in whole OpenZFS code base and it makes me wonder if there were reasons to use it, instead of `copy_from_user()`, like performance.

Reply:
There's no compelling reason I'm aware of, the intent here is to simply access all the user pages to make sure they're faulted in.